### ISSUE TITLE
Replace containerPath with Loader.request() - need to make sure it works though

### ISSUE THREAD

**Issue by sdeshpande3** [microsoft/FluidFramework#8274]

**Comment by ssimic2** [comment]
@scottn12 in FF repo we are simply removing containerPath altogether ? `Replace containerPath with Loader.request()` seems off, as (I thought) we are not replacing anything. Maybe some examples would be helpful to show how to use Loader.request() as a substitute.

**Comment by scottn12** [comment]
@ssimic2, that is correct, in the FF repo we are simply removing this API. We initially titled this task `Replace containerPath with Loader.request()` since we knew any usage of `containerPath` must be replaced with `Loader.request()`, but it turned out that it is only really used in partner repos, not the FF repo itself. Since we have broken down this item into multiple phases, we could rename this item to a more general title, for example `Remove containerPath from DriverPreCheckInfo`

**Comment by leeviana** [comment]
Update, I believe we have refactored our way out of the containerPath dependency using Loader.request() in our repo instead, so this can be safely removed :)
